src/IRTS/Bytecode.hs

```diff
@@ -113,11 +112,12 @@ data BC =
   deriving Show
 
 toBC :: (Name, SDecl) -> (Name, [BC])
-toBC (n, SFun n' args locs exp)
-   = (n, reserve locs ++ bc RVal exp True)
+toBC (nm, SFun _ _ locs e)
```

Comment:
Is it possible to use `_n'` and `_args` for the unused parameters? I find it hard to count plain underscores than those that are named.